### PR TITLE
[1LP][RFR]- fixing test

### PR DESCRIPTION
**Pull request by dgaikwad**
Fixed log in negative testcase

### PRT Run

{{pytest: cfme/tests/test_login.py::test_bad_password cfme/tests/automate/test_git_import.py::test_git_refresh_with_renamed_yaml cfme/tests/automate/test_domain.py::test_existing_domain_child_override --long-running -v}}

### CODE REVIEW

File: cfme/tests/test_login.py [hunk 1]
```diff
@@ -49,8 +49,6 @@ def test_login(context, method, appliance):
 @pytest.mark.tier(2)
 @pytest.mark.sauce
 @pytest.mark.parametrize('context', [ViaUI])
-# BZ 1632718 is only relevant for Chrome browser
-@pytest.mark.meta(blockers=[BZ(1632718)])
```
Comment (valaparthvi) [inline cfme/tests/test_login.py:53]: Can you add a `automates` marker and also add BZ description to polarion docstring? Even though it's only relevant to chrome browser, I think it's better to put it there.

Comment (dgaikwad) [reply cfme/tests/test_login.py:53]: We are getting the correct error message but not as per the BZ so I removed that BZ blocker and updated error message.

Comment (valaparthvi) [reply cfme/tests/test_login.py:53]: I'm not saying anything about the fix. It's just that this test was blocked by this BZ at some point in time which is why I think it is a good idea to keep a note of the BZ.

Comment (dgaikwad) [reply cfme/tests/test_login.py:53]: Done

**Pull request merged**